Assets/Mirror/Examples/Lobby/Scripts/PlayerController.cs

```diff
@@ -102,7 +101,6 @@ void OnControllerColliderHit(ControllerColliderHit hit)
         public void CmdClaimPrize(GameObject hitObject)
         {
             // Null check is required, otherwise close timing of multiple claims could throw a null ref.
-            //if (hitObject != null)
```

Comment:
please remove this comment completely if not needed anymore

Reply:
I removed the commented code...the comment stands as valid for the line below.